src/components/Icon/Icon.stories.tsx

```diff
@@ -55,3 +57,47 @@ export const Tones: StoryObj<typeof Icon> = {
   },
   argTypes: { tone: { control: false } },
 };
+
+export const Solids: StoryObj<typeof Icon> = {
```

Comment:
nit: 형용사를 복수화하는 것은 좀 어색한 영어 표현으로 느껴지네요.

```suggestion
export const Solid: StoryObj<typeof Icon> = {
```

Reply:
리뷰 감사합니다. 수정했습니다.